pages/docs/faqs.mdx

```diff
@@ -21,8 +21,17 @@ Once your Pull Request is merged it is deployed automatically. The deployment ca
 
 If your Profile is not visible after merging, please contact the maintainers in the <Link href="http://github.com/EddieHubCommunity/BioDrop/discussions">BioDrop GitHub Discussions</Link> repo.
 
+## My question is not answered by the FAQs/Where can I start a discussion about a feature or idea for BioDrop?
+
+BioDrop uses github discussions to have community conversations about anything related to the BioDrop project that does not fit into an issue. 
+
+An example of a github discussion could be proposing an idea for BioDrop that isn't ready for issue creation yet because it needs more details and input from the community.
```

Comment:
GitHub and Discussions should be capitalised

```suggestion
An example of a GitHub Discussion could be proposing an idea for BioDrop that isn't ready for issue creation yet because it needs more details and input from the community.
```